pippo-session-jdbc/README.md

```diff
@@ -0,0 +1,92 @@
+Pippo Session JDBC
+=====================
+[JDBC](https://docs.oracle.com/javase/tutorial/jdbc/) Session Manager integration for [Pippo](http://www.pippo.ro/)
+
+Sample code
+---------------
+
+Add the following code sniped in your application:
+
+```java
+public class MyApplication extends Application {
+
+    private DataSource dataSource;
+
+    @Override
+    protected void onInit() {
+        this.dataSource = createDataSource();
```

Comment:
I like this line. It is what I think. Inject an external `DataSource` in my application (via Spring, Guice, ...) or maybe create a new one via a factory method.